validator/sawtooth_validator/journal/journal.py

```diff
@@ -70,22 +70,34 @@ def stop(self):
             self._exit = True
 
     class _PublisherThread(Thread):
+        CHECK_PUBLISH_BLOCK_FREQUENCY = 0.1
```

Comment:
To clarify, I'm recommending this `def __init__(self, block_publisher, batch_queue, check_publish_block_frequency=0.1)`